### ISSUE TITLE
Deduplicate code by extending Django's SQLite backend

### ISSUE THREAD

**Issue by alanjds** [sum12/rqlite-django#1]
As lots of code seems to be copy-pasted from Django's SQLite backend, what about deduplicate it by extending instead of copy-pasting?

This way, bugfixes in SQLite upstream would propagate, and would be clearer what code is new and what is Django's one.

**Comment by sum12** [comment]
Yes, as you have noticed. It's just a copy with some fixes here and there.

I didn't want to invest time in keeping it clean unless the related PRs get
accepted.

So i will wait till then and clean this duplication.
